content/en/docs/reference/glossary/eviction.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Eviction
+id: eviction
+date: 2020-05-06
+full_link: /docs/tasks/administer-cluster/out-of-resource/
+short_description: >
+  The process of proactively failing one or more Pods on resource starved Nodes.
```

Comment:
I'm not sure I agree with this summary. An API client (eg an autoscaling controller) can decide that it wants to trigger Pod replacement by creating an Eviction request, as described in https://kubernetes.io/docs/tasks/administer-cluster/safely-drain-node/#the-eviction-api.

Evictions _usually_ happen in response to resource shortages, but that's not the only way they can happen.

Reply:
Oh right. My bad. In my limited experience with Kubernetes, I have only seen evictions in response to resource shortages. I am really going by - "The best way to learn something is to teach/(document) it." So apologies.
Let me spend some time to read up on the Eviction API and take another pass at this. If it seems too difficult I can unassign myself to let someone else with more experience tackle this one.
Are there guidelines available for how to write good glossary entries?